exercises/robot-simulator/canonical-data.json

```
@@ -294,10 +312,14 @@
       ]
     },
     {
-      "description": "Where R = Turn Right, L = Turn Left and A = Advance, the robot can follow a series of instructions and end up with the correct position and direction",
+      "description": "Series of instructions",
```

```suggestion
      "description": "Follow Left, Right and Advance instructions",
```

This seems like a fine movement from description to comment.

This means that this comment most likely does not end up with the user.

Comments are meant for track maintainers / test generator writers.

So maybe this means that a cryptic part of an exercise becomes more cryptic.

---

Are there any other instructions beside Left, Right, Advance?  Otherwise its redundant to mention them.